.gitignore

```diff
@@ -47,6 +47,7 @@
 !/**/*.R
 !/**/*.Rmd
 !/**/*.Rproj
+!renv.lock
```

Comment:
I forgot, but this actually has to be explicitly added in the top-level .gitignore so our list of packages can sync instead of the packages themselves

Reply:
Hi Tom, 
Do you mean that I need exactly copy this line to my gitignore?  The gitignore outside the renv folder?
Thanks for clarifying.

Reply:
Yeah, sorry, I forgot to mention this the other day. Or better yet, you could just copy the [entire .gitignore](https://github.com/worldbank/gld/blob/develop-merge/.gitignore) from `develop-merge` because there are many things added to account for the website

Reply:
I did the changes on my branch. thanks for bringing this up and feel free to close this issue for me if you wish.